Content.Server/UserInterface/ActivatableUISystem.cs

```diff
@@ -63,7 +82,7 @@ private bool InteractUI(EntityUid user, ActivatableUIComponent aui)
 
             if (aui.AdminOnly && !_adminManager.IsAdmin(actor.PlayerSession)) return false;
 
-            if (!_actionBlockerSystem.CanInteract(user))
+            if (!HasComp<GhostComponent>(user) && !_actionBlockerSystem.CanInteract(user))
```

Comment:
Shouldn't action blocker already check for ghostcomp? If not then we should add that.

Reply:
It does yeah. This is actively side stepping that for UI opening specifically so that spectators can open and view a UI but not interact with it. Though currently that requires them to use a verb, they can't just use 'E' (something something keybinding contexts?)